code/game/machinery/vending.dm

```diff
@@ -938,15 +938,16 @@
 		/obj/item/stack/medical/advanced/bruise_pack = 10,
 		/obj/item/stack/medical/advanced/ointment = 10,
 		/obj/item/stack/medical/splint = 5,
-		/obj/item/reagent_containers/hypospray/autoinjector/pain = 15
+		/obj/item/reagent_containers/hypospray/autoinjector/pain = 15, //tramadol autoinjector,
```

Comment:
why the comment?

Reply:
so people know what autoinjector/pain is for future reference.